liquidprompt

```diff
@@ -154,6 +156,35 @@ esac
 # CONFIGURATION #
 #################
 
+# Return true if the current Liquid Prompt version is greater than or equal to the specified version.
+# [OPTIONAL?] Return 1 if minor or less version difference, 2 if major difference.
+_lp_version_greatereq() {  # major, minor, [patch], [string], [number]
+    local major="$1" minor="$2" patch="${3-}" string="${4-}" number="${5-}"
+    if (( major > _LP_VERSION[0] )); then
```

Comment:
I forgot when writing my PoC, but this needs to use the `_LP_FIRST_INDEX` or whatever it is variable. That is probably why Zsh is failing tests.